src/ota.cpp

```diff
@@ -46,9 +46,12 @@ const char* u_host = "217.9.195.227";
 const char* u_url = "/esp/firmware.php";
 const char* firmware_update_path = "/upload";
 
+extern const char *esp_hostname;
+
 void ota_setup()
 {
   // Start local OTA update server
+  ArduinoOTA.setHostname(esp_hostname);
```

Comment:
What difference will this make? I haven't been using Arduino OTA therefore I'm probably unaware of any bugs here. Does this change fix or improve Arduino OTA? Have you been using Arduino OTA? Wondering if we should disable ArduinoOTA for production builds? I.e once we start using EmonESP for general use on OEM units.